doc/source/conf.py

```diff
@@ -121,7 +121,10 @@ def setup(app):
         "edit_link": False,
         "sidebar": "left",
         "scipy_org_logo": False,
-        "rootlinks": []
+        "rootlinks": [("https://numpy.org/", "NumPy.org"),
+                      ("https://numpy.org/doc", "Docs"),
```

Comment:
perhaps "Documentation" instead of "Docs"?

Reply:
hmm. This is part of a "breadcrumb" design across the top of the page, and can get quite wide as is, see for instance an [example page](https://8664-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/generated/numpy.subtract.html#numpy.subtract). I see I should put scipy.org first (or drop it altogether) to keep the breadcrumb order consistent. Let me know what you prefer: 
- docs vs documentation
- scipy.org first or not at all

Reply:
The [parallel page](https://docs.scipy.org/doc/numpy/reference/generated/numpy.subtract.html#numpy.subtract) on scipy.org uses `Docs`.

It also has a nice header, I wonder why the one from the [front page](https://www.numpy.org/) is not used in the numpy/doc and numpy/devdoc pages?

Reply:
ah I see, the example page helps. okay, let's stay with `Docs`.

> It also has a nice header, I wonder why the one from the front page is not used in the numpy/doc and numpy/devdoc pages?

not sure